data-tool/flows/tombstone/tombstone_mappings.py

```diff
@@ -197,6 +208,21 @@ def has_value(cls, value):
 
     EventFilings.FILE_CONVL: 'conversionLedger',
 
+    EventFilings.CONVAMAL_NULL: ['conversion', ('amalgamationApplication', 'unknown')],
+    EventFilings.CONVCIN_NULL: ['conversion', 'continuationIn'],
+    EventFilings.CONVCOUT_NULL: ['conversion', 'continuationOut'],  # TODO: continuation out
+    EventFilings.CONVDS_NULL: ['conversion', ('dissolution', 'voluntary')],
+    EventFilings.CONVDSF_NULL: ['conversion', ('dissolution', 'involuntary')],
+    EventFilings.CONVDSL_NULL: 'conversion',  # TODO: liquidation
+    EventFilings.CONVDSO_NULL: ['conversion', ('dissolution', 'unknown')],
+    EventFilings.CONVICORP_NULL: 'conversion',
```

Comment:
It's problematic to map `CONVICORP` to the behaviour of IA and update business info because `conv_event.effective_dt` is null for all. Similar issues for `CONVID1` and `CONVID2`, some don't have trigger_dt for involuntary dissolution stage transition.